pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -73,7 +73,7 @@ type GCECompute interface {
 	ListZones(ctx context.Context, region string) ([]string, error)
 	ListSnapshots(ctx context.Context, filter string, maxEntries int64, pageToken string) ([]*computev1.Snapshot, string, error)
 	GetSnapshot(ctx context.Context, project, snapshotName string) (*computev1.Snapshot, error)
-	CreateSnapshot(ctx context.Context, project string, volKey *meta.Key, snapshotName string) (*computev1.Snapshot, error)
+	CreateSnapshot(ctx context.Context, project string, volKey *meta.Key, snapshotName string, storageLocations string) (*computev1.Snapshot, error)
```

Comment:
I had a question about using an array but I checked that `parameters` is a `map[string]string` https://github.com/kubernetes-csi/external-snapshotter/blob/f255fe7b46bd5410370537319879d06f2bf1a156/client/apis/volumesnapshot/v1/types.go#L215, I don't know if users would do this though (NOTE the space between the comma and the other region):

```
parameters:
  storage-locations: us-central1, us-central2
```